Libplanet/Tx/TxSuccess.cs

```diff
@@ -41,7 +41,7 @@ public sealed class TxSuccess : TxExecution, ISerializable
             BlockHash blockHash,
             TxId txId,
             List<List<string>>? actionsLogsList,
-            IImmutableDictionary<Address, IValue?> updatedStates,
+            IImmutableDictionary<Address, IValue> updatedStates,
```

Comment:
Is it ok to code-level backward compatibility?

Reply:
This depends on whether `IAccountStateDelta.SetState()` accepted `null` or not.
- Although we are more stringent with `null`ables these days, if the current `interface` is to be believed, then that should have never been `null`.
- If we dig a little deeper, `ITrie.Set()` throws a `ArgumentNullException` when trying to set `null` as a value.

So, I'd say **probably**, but I'm not entirely sure. 😣